builder/vsphere/common/step_hardware.go

```diff
@@ -39,8 +39,10 @@ type HardwareConfig struct {
 	VGPUProfile string `mapstructure:"vgpu_profile"`
 	// Enable nested hardware virtualization for VM. Defaults to `false`.
 	NestedHV bool `mapstructure:"NestedHV"`
-	// Set the Firmware for virtual machine. Supported values: `bios`, `efi`, `efi-secure` or empty string to keep as in template. Defaults to empty string.
+	// Set the Firmware for virtual machine. Supported values: `bios`, `efi` or `efi-secure`. Defaults to `bios`.
```

Comment:
Could this be a breaking change ?

Reply:
No, because vsphere sets the default as `bios` and Packer just passes an empty string if this is not set. I just made the text more user friendly because `Defaults to empty string` makes vsphere defaults to Bios.

Reply:
Ah this makes sense! Is this the case for all versions of vsphere ?

Reply:
I would guess yes. The vm I'm building is compatible with ESXi 6.5 and later and vcenter ESXi 5.5 and later. I believe it wouldn't default to bios if it wasn't compatible with 6.5+ and according to https://github.com/hashicorp/packer/pull/9560 we will make sure to support 6.5 and later.

Reply:
Awesome !